omod/src/main/java/org/openmrs/module/fhir2/web/servlet/FhirRestServlet.java

```diff
@@ -35,10 +35,10 @@
 public class FhirRestServlet extends RestfulServer {
```

Comment:
Can you remove this file from this PR? Thanks!

Reply:
done 👍